ctapipe/utils/astro.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.coordinates import Angle
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+import logging
+
+logger = logging.getLogger(__name__)
+
+__all__ = ['get_bright_stars']
+
+
+def get_bright_stars(pointing=SkyCoord(ra=0. * u.rad, dec=0. * u.rad, frame='icrs'),
+                     radius=180. * u.deg, magnitude_cut=7.96):
```

Comment:
it might be clearer to have magnitude_cut default to `None`, so it clear all stars are included, rather than a "magic" value (though you could still note in the docstring that the Yale catalog stops at magnitude 7.96).   Should also mention which catalog is used in the docstring, ideally with a citation and date.

Reply:
changed and added in 96c2b77